fastlane/Fastfile

```diff
@@ -50,19 +54,22 @@ lane :upload_release do
 end
 
 desc 'Download the development signing certificates to this machine'
-lane :set_up_signing do
+lane :set_up_signing do |readonly: true|
+  require_env_vars!(*ASC_API_KEY_ENV_VARS, *CODE_SIGNING_STORAGE_ENV_VARS)
```

Comment:
You can see this in action in CI https://buildkite.com/automattic/hostmgr/builds/768/steps?jid=0196c1fa-00e8-4e72-93a4-9d021b0d49ca

<img width="785" alt="image" src="https://github.com/user-attachments/assets/62892653-5e2d-48c1-8fac-aa330b8eb74e" />

Reply:
And fixed by your recent commit in mobile-secrets (p1747011150457219-slack-C07DX5SGUVA) 👍